interface-definitions/system_watchdog.xml.in

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="system">
+    <children>
+      <node name="watchdog" owner="${vyos_conf_scripts_dir}/system_watchdog.py">
+        <properties>
+          <help>Hardware watchdog configuration</help>
+          <priority>9999</priority>
+        </properties>
+        <children>
+          <leafNode name="module">
+            <properties>
+              <help>Kernel module to load for watchdog device (optional)</help>
+              <valueHelp>
+                <format>txt</format>
+                <description>Module name (e.g. 'softdog', 'iTCO_wdt', 'sp5100_tco')</description>
```

Comment:
We generally prefer set-time validation over commit-time validation whenever possible, since it provides a faster feedback and the user doesn't have to commit to discover that some options are wrong. Two questions from this perspective:

1. How likely is it that the module for the hardware watchdog actually present in the system will not be loaded? Can we fully automate that and either load the module for the watchdog that the system has or fail the commit if there's no module matching the watchdog?
2. If we can't fully automate it, can we add an explicit list of available watchdog module for set-time validation?